app/api/profile.ts

```diff
@@ -77,7 +80,7 @@ class ProfileAPI extends PlutoAxios {
       is_current: params.isCurrent,
       institution_id: params.institutionId,
       institution_name: params.institutionName,
-      department: params.department,
+      department: !params.department ? null : params.department,
```

Comment:
When you use conditional ternary operator, please use a positive expression as much as possible for the condition.